sunpy/database/tests/test_tables.py

```diff
@@ -163,16 +168,16 @@ def test_entries_from_fido_search_result(fido_search_result):
     assert entries[60] == DatabaseEntry(
         source='SIDC', provider='SWPC', physobs='sunspot number',
         fileid='https://services.swpc.noaa.gov/json/solar-cycle/observed-solar-cycle-indices.json',
-        observation_time_start=datetime(2012, 1, 1, 0, 0),
-        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        observation_time_start=None,
+        observation_time_end=None,
```

Comment:
What happened to these datetimes?

Reply:
I killed them in the client. The noaa clients return results irrespective of date so it seemed insane we were putting the time of the query in the response.